automated install/basic-install.sh

```diff
@@ -351,40 +351,51 @@ package_manager_detect() {
         # Then check if dnf or yum is the package manager
         if is_command dnf ; then
             PKG_MANAGER="dnf"
-        else
+        elif is_command yum ; then
             PKG_MANAGER="yum"
+        elif is_command zypper ; then
```

Comment:
Please see this comment

https://github.com/pi-hole/pi-hole/pull/5027#discussion_r1046585261